content/influxdb/v1.8/tools/api.md

```diff
@@ -214,21 +214,21 @@ For more information about the Go `/net/http/pprof` package and the interactive
 
 #### `/debug/pprof/all` HTTP endpoint
 
-The `/debug/pprof/all` endpoint is a custom `/debug/pprof` profile intended primarily for use by InfluxData support. This endpoint generates a `profile.tar.gz` archive containing text files with the standard Go profiling information and additional debugging data. An optional CPU profile is generated when using the `cpu=true` option (default is `false`).
+The `/debug/pprof/all` endpoint is a custom `/debug/pprof` profile intended primarily for use by InfluxData support.
+This endpoint generates a `profile.tar.gz` archive containing text files with the standard Go profiling information and additional debugging data.
+An optional CPU profile is generated when using the `cpu=` option followed by a duration in seconds (e.g., `cpu=30s`).
 
 To create a `profile.tar.gz` archive, use the following cURL command to generate a `profile.tar.gz` file for sharing with InfluxData support.
 
 ```bash
-curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=true"
+curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=30s"
 ```
 
->**Note:** When the `cpu=true` option is included, a CPU profile is generated for 30+ seconds.
-> If you're concerned about running a CPU profile (which only has a small, temporary impact on performance), then you can set `?cpu=false` or omit `?cpu=true` altogether.
-
-As the following example shows, the cURL output includes "Time Spent," the time elapsed (in  seconds). After 30 seconds of data has been collected, the results are output to a file.
+As the following example shows, the cURL output includes "Time Spent," the time elapsed (in  seconds).
+After 30 seconds of data has been collected, the results are output to a file.
```

Comment:
Might want to add a note here that for 1.8.2 and earlier the correct command is the old one (`cpu=true`)

Reply:
1.8.2 and earlier, or 1.8.3 and earlier?

Reply:
I believe 1.8.3 and earlier - though nobody should be using 1.8.3 .

Reply:
Ah, okay 👍